src/host/services/IMUService.cpp

```diff
@@ -55,9 +56,13 @@ void IMUService::loop() {
   pitch = eulerAngles.y();
   heading = fmod(eulerAngles.x() + 270, 360);
 
-  if (sysCalib == 3) {
-    update.setNavigationAttitude(SKTypeAttitude(/* roll */ SKDegToRad(roll), /* pitch */ SKDegToRad(pitch), /* yaw */ 0));
+  if (magCalib >= cfHdgMinCal) {
     update.setNavigationHeadingMagnetic(SKDegToRad(heading));
     _skHub.publish(update);
+  }
+	
+	if (accelCalib >= cfHeelPitchMinCal && gyroCalib >= cfHeelPitchMinCal) {
```

Comment:
I think there is a indentation issue here. Can you make sure your editor is configured to replace tabs with spaces?

Reply:
Thanks, I was trying Netbeans as IDE because of it's internal versioning which I find great. Will have to take more care in configuring it.